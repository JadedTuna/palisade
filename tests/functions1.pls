fn proc(low x: int, high y: bool) high bool {
	
}

low y = proc(2, 3);