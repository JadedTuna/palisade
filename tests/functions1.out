[1;35msecurity error: [0minsecure explicit flow
   3 | }
   4 | 
   5 | low y [1;35m=[0m proc(2, 3);
       [1;35m~~~~~~[0m[1;35m^[0m
